dace/transformation/dataflow/map_for_loop.py

```diff
@@ -108,4 +109,7 @@ def replace_param(param):
         # Remove scope nodes
         nstate.remove_nodes_from([map_entry, map_exit])
 
+        # create outside hook
```

Comment:
What does hook mean? perhaps improve comment. Also, if it's meant to be accessed from the outside, why the leading underscore?